.github/workflows/main.yml

```diff
@@ -42,10 +42,17 @@ jobs:
       run: |
         pip install --no-binary :numpy: numpy==${{ matrix.numpy_ver }}
 
-    - name: Set up pysat
+    - name: Set up pysat on unix / mac
+      if: ${{ matrix.os != 'windows-latest'}}
       run: |
         mkdir pysatData
-        python set_ga_dir.py
+        python -c "import pysat; pysat.params['data_dirs'] = './pysatData'"
```

Comment:
at least on Mac pysat.params['data_dirs'] = 'pysatData'" works. Wouldn't need an os specific check if it also works on windows.